exporter/signalfxexporter/translation/dpfilters/datapoint.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dpfilters
+
+import (
+	"errors"
+
+	sfxpb "github.com/signalfx/com_signalfx_metrics_protobuf/model"
+)
+
+type dataPointFilter struct {
+	metricFilter     *stringFilter
+	dimensionsFilter *dimensionsFilter
```

Comment:
Are these pointers for performance reasons?

Reply:
Not really. I guess I just started out with them. Would you advocate for one vs the other?